lib/public/views/LhcFills/Overview/LhcFillsOverviewModel.js

```diff
@@ -75,10 +82,53 @@ export class LhcFillsOverviewModel extends OverviewPageModel {
      *
      * @return {boolean} true if the stable beams filter is active
      */
-    isStableBeamsOnly() {
+    getStableBeamsOnly() {
         return this._stableBeamsOnly;
     }
 
+    /**
+     * Returns all filtering, sorting and pagination settings to their default values
+     * @param {boolean} [fetch = true] whether to refetch all data after filters have been reset
+     * @return {void}
+     */
+    reset(fetch = true) {
+        super.reset();
+        this.resetFiltering(fetch);
+    }
+
+    /**
+     * Reset all filtering models
+     * @param {boolean} fetch Whether to refetch all data after filters have been reset
+     * @return {void}
+     */
+    resetFiltering(fetch = true) {
+        this._filteringModel.reset();
+
+        this._stableBeamsOnly = true;
```

Comment:
Once filtering is integrated with stableBeamsOnly, there will be no need for special case of stablebeams only (across the whole file)